### PR TITLE
Added agentless and no collector to word list

### PR DESCRIPTION
**Pull request by imatwawana**

### CODE REVIEW

File: docs/sources/write/style-guide/word-list/index.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ This page is a work in progress.
 
 ## A
 
+### agentless
+
+Don't use. Grafana Agent has been replaced by Grafana Alloy, so you shouldn't use agent-based terminology. See [no collector](#no-collector).
```
Comment (imatwawana) [inline docs/sources/write/style-guide/word-list/index.md:30]:
```suggestion
Don't use. Grafana Agent has been replaced by Grafana Alloy, so you shouldn't use agent-based terminology. Refer to [no collector](#no-collector).
```

**Review by chri2547 — APPROVED** [review]

**Review by josmperez — APPROVED** [review]
lgtm

**Comment by clayton-cornell** [comment]
The related Vale linting rule refers to it as `no-collector`.  Which is better/more correct here? `no collector` or `no-collector`?
https://github.com/grafana/writers-toolkit/pull/739

**Comment by imatwawana** [comment]
> The related [Vale linting rule PR](https://github.com/grafana/writers-toolkit/pull/739) refers to it as `no-collector`. Which is better/more correct here? `no collector` or `no-collector`?

Looks like from the Slack thread, the hyphen was the intention so I'll add it here.

**Review by clayton-cornell — APPROVED** [review]
LGTM :-)

**Pull request merged**